pkg/controllers/cluster/statefulset_control.go

```diff
@@ -29,12 +27,11 @@ import (
 )
 
 // StatefulSetControlInterface defines the interface that the
-// MySQLClusterController uses to create, update, and delete StatefulSets. It
+// MySQLClusterController uses to create and update StatefulSets. It
 // is implemented as an interface to enable testing.
 type StatefulSetControlInterface interface {
 	CreateStatefulSet(ss *apps.StatefulSet) error
```

Comment:
Ideally we'd rename this to alight with the name changes below i.e. 

statefulSetControl.Create(...)

Reply:
I’m actually planning a follow on to remove all the control interfaces and just use the client directly. With the fake client we don’t need the interfaces for testing and they just add a bunch of complexity and indirection.